include/enums.h

```diff
@@ -3307,39 +3307,61 @@ enum PlayerStatusFlags {
     PS_FLAGS_JUMPING                          = 0x00000002,
     PS_FLAGS_FALLING                          = 0x00000004,
     PS_FLAGS_FLYING                           = 0x00000008,
-    PS_FLAGS_10                               = 0x00000010,
-    PS_FLAGS_20                               = 0x00000020,
-    PS_FLAGS_40                               = 0x00000040,
-    PS_FLAGS_80                               = 0x00000080,
-    PS_FLAGS_100                              = 0x00000100,
-    PS_FLAGS_200                              = 0x00000200,
-    PS_FLAGS_400                              = 0x00000400,
-    PS_FLAGS_800                              = 0x00000800,
-    PS_FLAGS_1000                             = 0x00001000,
+    PS_FLAGS_SLIDING                          = 0x00000010,
+    /* Paused either via the start menu, or through another menu that causes a pause (like the item menu) */
+    PS_FLAGS_PAUSED                           = 0x00000020,
+    PS_FLAGS_CANT_CHANGE_PARTNER              = 0x00000040,
```

Comment:
Personally, I'd prefer something like "CHANGE_PARTNER_DISABLED" as opposed to cant

Reply:
Changed wording for these.